monitor_ingest/config.schema.yaml

```diff
@@ -0,0 +1,132 @@
+---
+action_type:
+  description: "Action type:  SetupEntityAction | SetupAddConstants | SetupAddDimesions | SetupAddFunctions "
+  type: "string"
+  required: false
+entity_name:
+  description: "Entity Name is must in case of action_type : LoadCsv"
+  type: "string"
+data_file_path:
+  description: "input data file path."
+  type: "string"
+  required: true
+json_schema_path:
+  description: "json Schema is must to validate CSV in case of action_type : DataClean"
```

Comment:
What is this a JSON schema for?

Reply:
The json schema defines particular format in which the data for ingestion should be supplied for our pack's data_ingest action

Reply:
Does `_path` refer to a file on disk?

And does this mean that the file will need to exist on all st2actionrunner nodes?

Reply:
Yes _path refers to a file on disk. What are you trying point with st2actionrunner nodes being using the path? Can you please explain?
Note - The _path is mandatory to execute setup_entity action.